garrysmod/lua/vgui/dlabelurl.lua

```diff
@@ -32,6 +30,17 @@ function PANEL:Init()
 end
 
 --[[---------------------------------------------------------
+	Paint
+-----------------------------------------------------------]]
+function PANEL:Paint()
+
+	-- Some trickery to hide the DButton
+	surface.SetDrawColor(0,0,0,0)
+	surface.DrawRect( 0, 0, self:GetWide(), self:GetTall() )

```

Comment:
Something tells me that leaving this function completely empty will achieve the same effect as drawing an invisible rectangle.